.github/workflows/releaseFail.yml

```diff
@@ -0,0 +1,44 @@
+# test
+on:
+  workflow_run:
+    workflows:
+      - RC
+      - promote-rc-to-latest
+      - manual promote
+      - pr for rc
+      - rc tag/release
+    types:
+      - completed
+
+jobs:
+  failure-notify:
+    runs-on: ubuntu-latest
+    if: ${{ github.event.workflow_run.conclusion == 'failure' }}
+    steps:
+      - name: Announce Failure
+        id: slack
+        uses: slackapi/slack-github-action@v1.21.0
+        env:
+          SLACK_WEBHOOK_URL: ${{ secrets.CLI_ALERTS_SLACK_WEBHOOK }}
+          SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+        with:
+          payload: |
+            {
+              "text": "Release failed: ${{ github.event.workflow_run.repository.name }}",
+              "blocks": [
+                {
+                  "type": "header",
+                  "text": {
+                    "type": "plain_text",
+                    "text": "Release failed: ${{ github.event.workflow_run.repository.name }}"
```

Comment:
Let's wrap this line with `:attentioncse:` (or something similar) so it stands out

Reply:
Can you get the name of the workflow that failed? "Release failed" does not apply to some of the workflows we are watching

Reply:
see `workflow name` below.  I can put that in the title, too.